### PR TITLE
Add tabby with auto updates

### PR DESCRIPTION
**Pull request by Jai-JAP**
Tabby is A terminal for the modern age
https://tabby.sh
https://github.com/Eugeny/tabby
Arm builds hosted [here](https://github.com/Jai-JAP/tabby-arm-builds)

### CODE REVIEW

File: apps/Tabby/description [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ A terminal for a more modern age
```
Comment (Botspot) [inline apps/Tabby/description:1]: Try to be more descriptive and explain why it's better than alacritty, eDEX-UI, Hyper, lxterminal, guake, etc.

**Comment by Jai-JAP** [comment]
```
git clone https://github.com/eugeny/tabby
cd tabby
curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
sudo apt install -y libsecret-1-dev libfontconfig1-dev libarchive-tools nodejs
sudo npm i -g yarn
```

**Comment by theofficialgman** [comment]
is there a reason there isn't an armhf build of this?
I should also note, make sure you built on buster, otherwise the higher glibc of bullseye version will cause issues for electron on buster

**Comment by ryanfortner** [comment]
Works great on RPiOS 64-bit bullseye. 
image

**Comment by Jai-JAP** [comment]
> 

This is built on rpios buster lite

**Comment by theofficialgman** [comment]
> Tabby doesn't seem to support armhf as
> 
> * Electron >= 17.0.1 (v8 engine) uses compressed pointers which is a 64 bit only feature
> * Even if electron is downgraded to 17.0.0 it refuses to build saying arch arm is not supported
> * Also CI builds are not possible as cross compiling doesn't work for some reason and results in x86_64 node libs instead of arm64
> 
> See [Eugeny/tabby#5907](https://github.com/Eugeny/tabby/pull/5907)
> 
> This is built on rpios buster lite

huh... well I know at least building for armhf on 17.0.0 is possible, as that is what version of electron webcord uses.
@Jai-JAP I added a potential solution at the PR

**Comment by Jai-JAP** [comment]
Thanks to @theofficialgman and @SpacingBat3 armv7l build was successful with electron 17.0.0 added it to the scripts

**Comment by theofficialgman** [comment]
I gave up on trying to fix the pi-apps logo a long time ago. This is what it looks like on my system RN in gnome terminal

Screenshot from 2022-03-25 12-24-58

**Comment by Botspot** [comment]
> I gave up on trying to fix the pi-apps logo a long time ago. This is what it looks like on my system RN in gnome terminal

Could you return the output of `grep 'VERSION_ID=' /etc/os-release | tr -cd '0123456789'`

**Comment by theofficialgman** [comment]
it `1804` like you would expect

**Comment by theofficialgman** [comment]
```
NAME="Ubuntu"
VERSION="18.04.6 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.6 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

**Comment by Jai-JAP** [comment]
This is now ready to merge

**Pull request merged**